code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -224,6 +224,10 @@ GLOBAL_LIST_EMPTY(PDAs)
 		to_chat(user, span_warning("You don't have the dexterity to do this!"))
 		return
 
+	if(HAS_TRAIT(src, TRAIT_PDA_MESSAGE_MENU_RIGGED) && mode == 2)
```

Comment:
what the fuck is 2

Reply:
One of the many magic numbers for the mode of the pda UI. I'll fix em soon.